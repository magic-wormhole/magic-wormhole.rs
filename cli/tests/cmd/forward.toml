bin.name = "wormhole-rs"
args = "forward"
status.code = 2
