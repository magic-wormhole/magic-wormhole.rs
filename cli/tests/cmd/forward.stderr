Forward ports from one machine to another

Usage: wormhole-rs[EXE] forward [OPTIONS] <COMMAND>

Commands:
  serve[..]
  connect[..]

Options:
...
